pinot-broker/src/main/java/org/apache/pinot/broker/api/RequestStatistics.java

```diff
@@ -220,6 +222,10 @@ public long getOfflineThreadCpuTimeNs() {
     return _offlineThreadCpuTimeNs;
   }
 
+  public int getNumRowsResultSet() { return _numRowsResultSet; }
+
+  public void setNumRowsResultSet(int numRowsResultSet) { _numRowsResultSet = numRowsResultSet; }
```

Comment:
Can we remove this method as no one seems to be calling this ?

Reply:
Most of the getters and setters are used here, should we add a follow up to clean them as well

Reply:
updated!

Reply:
We still need the getter as we will use it internally